### ISSUE TITLE
Turnigy Multistar 45A - update board pictures

### ISSUE THREAD

**Issue by ycherniavskyi** [sim-/tgy#55]
Hi,
I have noticed that links to board pictures of Turnigy Multistar 45A in https://docs.google.com/spreadsheet/ccc?key=0AhR02IDNb7_MdEhfVjk3MkRHVzhKdjU1YzdBQkZZRlE are broken.

I decide to fix this issue and shoot my Turnigy Multistar 45 Amp Multi-rotor Brushless ESC 2-6S (OPTO) with PRODUCT ID: 9351000009. I think it could help someone.

![9351000009_cover](https://cloud.githubusercontent.com/assets/552025/3708709/75973b30-1447-11e4-8506-ff0f128a5446.jpg)

![9351000009_top](https://cloud.githubusercontent.com/assets/552025/3708715/7e669012-1447-11e4-9665-c91b6cda2dbd.jpg)

![9351000009_back](https://cloud.githubusercontent.com/assets/552025/3708718/86273a54-1447-11e4-8e0d-da6505ce6470.jpg)

![9351000009_perspective](https://cloud.githubusercontent.com/assets/552025/3708721/8ca59b0a-1447-11e4-865d-7d9833cd0477.jpg)

**Comment by idzuwan** [comment]
So I guessing that same firmware are used for both opto and bec version?

**Comment by ycherniavsky** [comment]
Yes, I used `dlu40a.hex`. Unfortunately I have not provide any real load test of it. On dest it works as expected.

**Comment by idzuwan** [comment]
cool, I gonna flash mine and do some flying tomorrow or the next day, with stock firmware esc keep getting out of sync with 360kv motor while doing autotune with Arducopter
